apps/src/templates/census2017/CensusInaccuracyReview.jsx

```diff
@@ -0,0 +1,400 @@
+
+    return (
+      <div>
+        <h1 id="report-header">
+          Reported Access Report Inaccuracies
+        </h1>
+        {display}
+      </div>
+    );
+  }
+
+  stringForExplanationLabel = {
+    offers_ap: "Offers AP CS?",
+    offers_ib: "Offers IB CS?",
+    consistent_teacher_surveys: "Consistent Teacher/Admin Surveys",
+    state_offering: "Teaches per State Data?",
+    consistent_non_teacher_surveys: "Consistent Non-Teacher/Admin Surveys",
+    inconsistent_surveys: "Inconsistent Surveys",
+    last_years_summary: "Last Year's Summary",
+    two_years_agos_summary: "Two Years Ago's Summary",
+  };
+
+  renderReview = () => {
+    const noData = 'No Data';
+    const school = this.state.toReview.school;
+    const auditData = this.state.toReview.summary_audit;
+    const explanation = auditData.explanation;
+    const highlightedStyle = {
+      backgroundColor: "#8afc9b",
+    };
+
+    const overrideText = `Create a ${this.state.toReview.teaches_cs ? 'YES' : 'NO'} override for this school.`;
+    return (
+      <div style={{maxWidth: 615}}>
+        <h2>
+          {school.name} ({school.city}, {school.state})
+        </h2>
+        <Button
+          onClick={this.showMainList}
+          size="large"
+          text="Back to main list"
+        />
+        <br />
+        <hr />
+        <div>
+          <h3>We currently classify this school as <b>{auditData.previous_years_results[0] || 'NULL'}</b> based on:</h3>
+          <table className="table table-bordered table-striped">
+            <tbody>
+              {explanation.map((data, index) => {
+                 let value;
+                 if (data.value === null) {
+                   value = noData;
+                 } else if (data.value instanceof Object) {
+                   if (data.value.yes === 0 && data.value.no === 0) {
+                     value = noData;
+                   } else {
+                     value = `${data.value.yes} Yes / ${data.value.no} No`;
+                   }
+                 } else if (typeof(data.value) === "boolean") {
+                   value = data.value ? 'Yes' : 'No';
+                 } else {
+                   value = data.value;
+                 }
+
+                 return (
+                   <tr key={index}>
+                     <td style={data.used ? highlightedStyle : null}>
+                       {this.stringForExplanationLabel[data.label]}
+                     </td>
+                     <td style={data.used ? highlightedStyle : null}>
+                       {value}
+                     </td>
+                   </tr>
+                 );
+              }
+              )}
+            </tbody>
+          </table>
+          The highlighted row was used to determine if this school teaches CS.
+        </div>
+        <br />
+        <hr />
+        <div>
+          <h3>Inaccuracy Report Data:</h3>
+          <table className="table table-bordered table-striped">
+            <tbody>
+              <tr>
```

Comment:
It feels like there might be a loop that could generate all these columns instead of hardcoding them.

Reply:
I changed this to use reactabular.